stories/components/Modals.js

```diff
@@ -541,5 +543,78 @@ storiesOf('Components', module).add('Modals', () => (
                 </Modal.Container>
             </ModalTrigger>
         </StoryItem>
+        <StoryItem
+          title="SelectionModal"
+          description="SelectionModal can be used with other Modal components to create a selection layout"
+        >
+          <ModalTrigger>
+            <SelectionModal size="large">
+              <Modal.Header useTitle={false}>
+                <h3 className="text-lg mt-2 mb-3">Import templates</h3>
+                <p className="m-0 leading-6">Imported templates will be duplicated, meaning changes made to the originals won’t be reflected here. Any components will also be imported in the process.</p>
+              </Modal.Header>
+              <SelectionModal.Body>
+                <SelectionModal.Column isHighlight className="w-3/6 flex-shrink-0 px-4 pb-8">
+                  <SelectionModal.ColumnHeader className="mb-3">PROJECTS</SelectionModal.ColumnHeader>
```

Comment:
💚 Should any of the classes we have added via the `className` prop be baked into the component?

They may not, but it's worth I thought! It would be cool if we just put the components together and they are styled without external jiggery - that said, this may not be sensible!

Reply:
I went back and forth on this and landed on no because in the future we may want the columns a different size.

Reply:
Great - sounds good!